src/widgets/survey/model/services/tests/testHelpers.ts

```diff
@@ -232,6 +233,7 @@ export const getActivityProgressMock = (): StoreProgress => {
           availableTo: null,
           startAt: 12367800000,
           endAt: null,
+          entityName: 'mock-entity-name',
```

Comment:
Change it to empty string, please. The related BL doesn't use this field. And if even so, it'd be easier to catch. 